flexmeasures/data/schemas/account.py

```diff
@@ -35,6 +36,21 @@ class Meta:
     account_roles = fields.Nested("AccountRoleSchema", exclude=("accounts",), many=True)
     consultancy_account_id = ma.auto_field()
 
+    @validates("primary_color")
+    def validate_primary_color(self, value):
+        if value is not None and not validate_color_hex(None, "primary_color", value):
+            raise ValidationError("Invalid HEX color for primary_color.")
+
+    @validates("secondary_color")
+    def validate_secondary_color(self, value):
+        if value is not None and not validate_color_hex(None, "secondary_color", value):
```

Comment:
Refactor request: `validate_color_hex()` doesn't need the `ctx` parameter anymore.
I would also not send it the parameter name. It's job is only to say True or False, and not to print out anything.

Reply:
I'm actually using this same function for the cli command, to avoid having two functions doing the same thing I made some modifications so it can be used by the cli and also anywhere else

Reply:
That's great, but a simple util like this should concentrate one one thing. 
For the message, in the schema you make a different one anyway

Reply:
That's great, but a simple util like this should concentrate one one thing. 
For the message, in the schema you make a different one anyway

Reply:
Sorry, I'm not quite sure what you mean. This is specifically focused on validating whether a color is a valid hex value.

Reply:
Would you suggest splitting it into two functions—one for CLI and another for other uses?